views/templates/front/payment.tpl

```diff
@@ -225,7 +227,7 @@
                             type: "POST",
                             url: paymentProcessUrl,
                             data: data,
-                            dataType: "json",
+                            dataType: "text json",
```

Comment:
Is there any case when we don't return json back but text?

Reply:
The server response type is `text/html`. According to the jQuery docs, adding `text json` tells the library that the response needs to be converted from plain text to JSON. Adding this solved an issue with the 3DS2 modal for me.